src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -143,7 +145,21 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
             throw new EntityDoesNotExistException(
                     "Trying to get feedback sessions for a course that does not exist.");
         }
+        return getFeedbackSessionsForUserInCourseSkipCheck(courseId, userEmail);
+    }
 
+    /**
+     * Under the reasonable assumption (or after verifying) that the course <b>courseId</b>
+     * exists, this method gets the feedback sessions for the specified user in the course.
+     * 
+     * @param courseId
+     * @param userEmail
+     * @return a list of viewable feedback sessions for any user for his course.
+     * @throws EntityDoesNotExistException
```

Comment:
Managed to fix the code so that EDNEE won't be thrown (although, really EDNEE won't be thrown, it's there because of the logic methods that are called further down). Also, as a beneficial side effect, I managed to suppress one more source of redundant query.